src/common/api/orcidlinkAPI.ts

```diff
@@ -1,56 +1,79 @@
 import { baseApi } from '.';
-import { jsonRpcService } from './utils/serviceHelpers';
+import { LinkRecordPublic, ORCIDProfile } from './orcidLinkCommon';
+import { jsonRpc2Service } from './utils/serviceHelpers';
 
-// orcidlink system types
+// is-linked
+
+export type IsLinkedResult = boolean;
```

Comment:
I often use this style because to me It helps keep the api implementation consistent - one expects a parameter type and a result type named in a specific manner. In fact, I typically create the api method, with named types, and then back fill the type definitions. I tend not to go back and remove a type like this if it resolves down to a simple type alias.
That said, I know that some prefer not to use a type alias like this; I can easily change it.

Reply:
I did change this